extras/wait-for-bricks-test.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash -x
```

Comment:
Should we disable trace output (-x) ? or is this intentional ?

Reply:
Also should we move the test script under tests/ dir ?

Reply:
It is better for the test to give as much information as possible, so it seemed okay. This test is not testing functionality of gluster-block and testing something in extras. So I kept it that way.